src/servers/sapphire_zone/ServerMgr.h

```diff
@@ -59,6 +59,7 @@ class ServerMgr
 
   std::map< uint32_t, SessionPtr > m_sessionMapById;
   std::map< std::string, SessionPtr > m_sessionMapByName;
+  std::map< uint32_t, std::string > m_payerNameMap;
```

Comment:
`m_playerNameMap`

Reply:
Name should also indicate what is mapped to the name. See the other maps